index.html

```diff
@@ -137,6 +137,12 @@ <h2>Related Projects</h2>
                                 <li><a href="https://github.com/cadorn/ace-extjs">Ace wrapper for ExtJS</a></li>
                                 <li><a href="https://github.com/daveho/AceGWT">Ace wrapper for GWT</a></li>
                             </ul>
+                            <h2>Getting Help</h2>
+                            <ul class="content-list">
+                                <li><a href="https://github.com/ajaxorg/ace/issues">Report bugs</a></li>
+                                <li><a href=" https://github.com/ajaxorg/ace/discussions">Discuss</a></li>
+                                <li><a href="https://stackoverflow.com/questions/tagged/ace-editor">Ask a qestion on StackOverflow</a></li>
```

Comment:
typo `qestion` -> `question`

Reply:
lets rename it to something like `You can also engage with our community on StackOverflow.` and move it out of the list to a separate line - I'd suggest to keep GitHub links as the main source of assistance, and keep link to stackoverflow separate to not associate it with GitHub links.

Reply:
I've removed it to not add too much complexity, stackoverflow is active enough anyway.